### PR TITLE
Eng 1803 add trivial accept

### PR DESCRIPTION
**Pull request by texasmichelle**
Skips API prediction calls for all trivial-accept tasks

### CODE REVIEW

**Comment by texasmichelle** [comment]
@idigary I think I covered everything in your comments. Anything else look odd?

File: java-api-client/src/test/java/com/idibon/api/model/TaskTest.java [hunk 1]
```diff
@@ -57,4 +59,76 @@
...
+        assert(predictionJson.getJsonObject(0).get("features") == null);
+        
+        // Specify that we want to see features, then ensure that an empty feature object is included
+        prediction = prediction.withSignificantFeatures();
+        predictionJson = prediction.iterator().next().right.getJson();
+        assert(predictionJson.getJsonObject(0).get("features").toString().equals("{}"));
+    }
+    
+    @Test public void testNonTrivialRules() throws Exception {
+        String json = "{\"task\":{\"uuid\":\"00000000-0000-0000-0000-000000000000\",\"name\":\"ClassifyCats\"," +
```
Comment (idigary) [inline java-api-client/src/test/java/com/idibon/api/model/TaskTest.java]: I'd like to see one additional test case here -- a task that just uses ClarabridgeRules, where one label is non-trivial.

**Comment by idigary** [comment]
this looks good -- I'd like one more negative test case added, but everything else is :+1:

**Comment by texasmichelle** [comment]
I added a 2nd nontrivial test that looks exactly like the JSON for the trivial case, with one exception: 'white' is present in the label_rule for Persian. Is that what you meant?

**Comment by idigary** [comment]
yes, that's exactly what I meant.

**Comment by texasmichelle** [comment]
@idigary - ready for your perusal.

**Comment by texasmichelle** [comment]
Great catch! Removed all the awkwardness. Or some of it anyway.

**Comment by idigary** [comment]
:ship: it

**Pull request merged**